modules/osdk-csv-annotations-infra.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="osdk-csv-annotations-infra_{context}"]
+= Infrastructure features
+
+The following Operator annotations detail the infrastructure features that an Operator might support, denoted with a `true` or `false` value. Users can view and filter by these features when discovering Operators through OperatorHub in the web console or on the link:https://catalog.redhat.com/software/search?deployed_as=Operator[Red Hat Ecosystem Catalog].
+
+[NOTE]
+====
+The following infrastructure feature annotations are supported in {product-title} 4.10 and later.
+====
+
+.Infrastructure features annotations
+[cols="4a,5a,options="header"]
+|===
+|Annotation |Description
+
+|`features.operators.openshift.io/disconnected`
+|Whether an Operator leverages the `spec.relatedImages` CSV field and can run without an internet connection by referring to any related image by its digest.
```

Comment:
```suggestion
|Specifies whether an Operator leverages the `spec.relatedImages` CSV field and can run without an internet connection by referring to any related image by its digest.
```
Consider adding `Specifies` (or some similar word) before `whether` to make the phrase more verb-centric in all of these descriptions, and the change would make the wording here parallel the grammatical structures of the other table's description later in this PR.